src/library/basesqltablemodel.cpp

```diff
@@ -43,10 +43,7 @@ BaseSqlTableModel::BaseSqlTableModel(
         QObject* parent,
         TrackCollectionManager* pTrackCollectionManager,
         const char* settingsNamespace)
-        : BaseTrackTableModel(
-                  settingsNamespace,
-                  pTrackCollectionManager,
-                  parent),
+        : BaseTrackTableModel(parent, pTrackCollectionManager, settingsNamespace),
```

Comment:
Why did you reformat this code? Both formats are valid.

Keeping parameters of long parameter or argument lists on separate lines allows to easily review changes. In general long parameter lists should be avoided.

Reply:
I agree, and unnecessary reformatting should generally be avoided. In this case we can still merge it either way because the resulting line is still short enough IMHO.

Reply:
Because I wanted to maintain a consistent style throughout the whole class hierarchy. And since it is merely passing all arguments up that are given to it, which are already listed on separate lines above, I don't think that it is worth to list them all on separate lines again.